api/integrations/fifabot/package-lock.json

```diff
@@ -0,0 +1,3217 @@
+{
```

Comment:
Well, two things:

1 - I doubt you use all of this as the dependencies for the bot
2 - You're using stuff from the Symphony repo which I don't think you should use.